matrix-seshat not found

When starting riot-web, I get:
```
seshat unavailable Error: Cannot find module 'matrix-seshat'
Require stack:
- /opt/Riot/resources/app.asar/src/electron-main.js
- 
    at Module._resolveFilename (internal/modules/cjs/loader.js:717:15)
    at Function../lib/common/reset-search-paths.ts.Module._resolveFilename (electron/js2c/browser_init.js:7609:16)
    at Module._load (internal/modules/cjs/loader.js:622:27)
    at Module._load (electron/js2c/asar.js:717:26)
    at Function.Module._load (electron/js2c/asar.js:717:26)
    at Module.require (internal/modules/cjs/loader.js:775:19)
    at require (internal/modules/cjs/helpers.js:68:18)
    at Object.<anonymous> (/opt/Riot/resources/app.asar/src/electron-main.js:49:14)
    at Module._compile (internal/modules/cjs/loader.js:880:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:892:10) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [ '/opt/Riot/resources/app.asar/src/electron-main.js', undefined ]
}
```

Using the riot-web 1.5.6 .deb package.

Everything seems to work at first sight.

--- a/electron_app/src/electron-main.js
+++ b/electron_app/src/electron-main.js
@@ -47,7 +47,11 @@ let Seshat = null;
 try {
     Seshat = require('matrix-seshat');
 } catch (e) {
-    console.warn("seshat unavailable", e);
+    if (e.code === "MODULE_NOT_FOUND") {
+        console.log("Seshat isn't installed, event indexing is disabled.");
+    } else {
+        console.warn("Seshat unexpected error:", e);
+    }
 }
 
 if (argv["help"]) {

